doc-utils/component-example.scss

```diff
@@ -66,7 +66,7 @@
             margin-top: jkl.$spacing-l;
         }
         @include jkl.text-style("small");
-        color: jkl.$color-fjellgra;
+        color: unset;
```

Comment:
Funker det å fjerne `color` helt? Ingenting som er bedre enn å få fjernet kode hvis man kan 😁

Reply:
Teksten ble sort da jeg fjernet det via devtools i Safari, på forhåndsvisnings-URLen. Virker som den falt tilbake til default browserstil.

`unset` funker fint for min del :)

Reply:
> Funker det å fjerne `color` helt? Ingenting som er bedre enn å få fjernet kode hvis man kan 😁

Done :)